packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx

```diff
@@ -204,8 +204,8 @@ const adminSettings = () => {
                 </div>
               </div>
               <div className={classes.floatright}>
-                <Link to="/admin/delete">
-                  <button>{t("delwork_manage")}</button>
+                <Link to="/admin/workspace/delete">
+                  <button>Delete Workspace</button>
```

Comment:
Someone adds translation to the app and you remove it? Isn't this counter productive. If you don't understand something just leave it there

Reply:
I will change it, that was someone merge stuff to counter my work which i added earlier, so i had to keep what i was working with earlier, by the way, this button wasn't there I'm the one that added it, only for me to pull and find out someone somewhere is also changing stuff.

Reply:
Well, translation for all pages is ongoing. When resoling conflicts, you accept both changes and keep the changes from the other person and add the ones you're working on. Not removing the other person's work entirely. The only instance when you can delete is if that functionality isn't needed anymore